### DISCUSSION TITLE
Pest test files not running during testing

### DISCUSSION THREAD

**Discussion by jbdev21** [pestphp/pest#538] [Q&A]
I have an issue wherein most of my pest test files are not running during running the test. It will work if I run it individually using the VS code but it will not run if I use the command ``` php artisan test ``` or  ``` ./vendor/bin/pest ```. 

I have both PHPUnit syntax tests and Pest syntax tests all together since I am just converting it all to pest. If I run the test only the PHPUnit syntax files are running. I tried to find a solution but I really can't find it and tried some suggestions but it is still not working. I really appreciate your ideas and help. Thank you in advance.

**Comment by lukeraymonddowning** [comment]
Mind posting your base TestCase and the output when you run "vendor/bin/pest"?

**Reply by jbdev21** [reply]
![image](https://user-images.githubusercontent.com/29841815/178073696-471cccdd-779b-4f5f-8c5c-2cfeb0f3c14a.png)
here is the code in my Pest.php

![Untitled](https://user-images.githubusercontent.com/29841815/178073881-5f45dfbd-637e-4efe-80e9-45531aac2a1e.png)
this is the code in my Tests\TestCase.php

**Comment by jbdev21** [comment]
if I run ``` ./vendor/bin/pest ``` only PHPUnit syntax tests file is running. I also tried to remove all the PHPUnit tests for a moment and run the test and nothing is running at all. In short, Pest test files is not firing up during my test.

**Comment by fabio-ivona** [comment]
@jbdev21 would you mind to check if you have a ->only() call in your tests?

**Reply by Procionegobbo** [reply]
> @jbdev21 would you mind to check if you have a ->only() call in your tests?

thanks @fabio-ivona you just cured my headache
